Description: Remove com.google.common.util.concurrent.internal

 com.google.common.util.concurrent.internal is copied from futures/failureaccess
 to src before building and so it is not available to be Imported as a separate
 package. Remove it from the manifest.

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Forwarded: not-needed

---

--- guava-libraries-29.0.orig/guava/pom.xml
+++ guava-libraries-29.0/guava/pom.xml
@@ -79,7 +79,6 @@
               com.google.common.*
             </Export-Package>
             <Import-Package>
-              com.google.common.util.concurrent.internal,
               javax.annotation;resolution:=optional,
               javax.crypto.*;resolution:=optional,
               sun.misc.*;resolution:=optional
